pages/news-and-events/index.jsx

```diff
@@ -27,37 +27,75 @@ export default function NewsAndEventsPage({ previews }) {
-            slug,
-          } = preview;
-
-          return (
-            <article key={title} className="mb-12 break-words">
-              <h2>
-                <Link legacyBehavior href={`/news-and-events/${slug}`}>
-                  <a className="normal-case text-samGreyDark hover:text-samDarkRed">
-                    {title}
-                  </a>
-                </Link>
-              </h2>
-              <NewsMeta categories={categories} date={date} />
-              <MarkdownContent content={excerpt} />
-            </article>
-          );
-        })}
+
+        <div className="py-12 bg-white">
+          <div className="px-6 mx-auto max-w-7xl lg:px-8">
+            <div className="max-w-2xl mx-auto text-center">
+              <h1 className="text-3xl font-bold tracking-tight text-gray-900 sm:text-4xl">
+                News and Events
+              </h1>
+              <p className="mt-2 text-lg leading-8 text-gray-600">
+                Learn about the latest News and Events in the Samvera Community.
+              </p>
+            </div>
+            <div className="grid max-w-2xl grid-cols-1 mx-auto mt-16 gap-x-8 gap-y-20 lg:mx-0 lg:max-w-none lg:grid-cols-3">
+              {/* Blog Posts from the CMS */}
+              {blogPosts.map((post) => {
+                const {
+                  content,
+                  image,
+                  publishDate,
+                  slug = "#",
+                  tag = [],
+                  title,
+                } = post.fields;
+                return (
+                  <BlogPostPreview
+                    key={post.sys.id}
+                    cmsContent={content}
+                    image={image}
+                    publishDate={formatDateString(publishDate)}
+                    slug={slug}
+                    tag={tag}
+                    title={title}
+                  />
+                );
+              })}
+
+              {/* Blog Posts from local markdown files */}
+              {previews.map((preview, index) => {
+                const {
+                  excerpt,
+                  frontmatter: { categories, date, title },
+                  slug,
+                } = preview;
+
+                return (
+                  <BlogPostPreview
+                    key={index}
+                    markdownContent={excerpt}
+                    publishDate={formatDateString(date)}
+                    slug={slug}
+                    tag={categories}
+                    title={title}
+                  />
+                );
+              })}
+            </div>
+          </div>
+        </div>
       </Main>
     </Layout>
   );
 }
 
 export async function getStaticProps() {
   const { previews } = getNewsPreviews();
+  const subsetOfPreviews = previews.slice(0, 30);
+
+  const blogPosts = await getBlogPosts();
 
   return {
-    props: { previews },
+    props: { blogPosts, previews: subsetOfPreviews },
```

Comment:
```suggestion
    props: { blogPosts, previews: subsetOfPreviews, openGraphData },
```

Since this is the coming as index.jsx page, perhaps we should hardset these openGraphData values as well? Otherwise it seems to fall back to homepage values.